internal/xds/translator/translator.go

```diff
@@ -34,14 +38,14 @@ type GlobalRateLimitSettings struct {
 }
 
 // Translate translates the XDS IR into xDS resources
-func (t *Translator) Translate(ir *ir.Xds) (*types.ResourceVersionTable, error) {
+func (t *Translator) Translate(ir *ir.Xds, extManager extensionTypes.Manager) (*types.ResourceVersionTable, error) {
```

Comment:
you probably want `extensionTypes.Manager` to be set within the `Translator` context instead of an arg